index.html

```diff
@@ -0,0 +1,350 @@
+            </figcaption>
+          </figure>
+        </section>
+        <section class="clients">
+          <h3 class="client-section-header gray-line">CLIENTS</h3>
+          <button class="icon-chevron-left client-button circle"></button>
+          <ul>
+            <li>
+              <figure class="client-figure">
+                <img src="/img/broderic.jpg" alt="Image of Matthew Broderic" title="Matthew Broderic" class="client-image circle">
+                <figcaption class="client-caption">
+                  <h4 class="team-header client-header"><span class="client-name">Matthew Broderic</span> (CEO Hopicola Inc.)</h4>
+                  <p class="team-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard my text ever since the 1500s, when lorem upsum is the best text for you.</p>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
+          <button class="icon-chevron-right client-button circle"></button>
+        </section>
+      </article>
+      <article id="contact" class="contact-section section-arrows-white">
+        <h2 class="contact-section-header section-header white-color white-line">CONTACT</h2>
+        <p class="contact-text">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <form class="contact-form">
+          <input type="text" name="company-name" placeholder="Company name">
+          <input type="email" name="company-email" placeholder="Your@mail.com">
+          <textarea name="company-message" rows="8" cols="60">Message...</textarea>
+          <input type="submit" name="submit-button" value="SEND MESSAGE">
+        </form>
+        <figure class="contact-map">
+          <img src="/img/map-mobile.jpg" alt="Map of San Francisco location" title="San Francisco location">
+          <figcaption class="contact-map-caption">
+            <a class="contact-phone" href="tel:970897580">+970 897 580 <span class="contact-weight-light">(7:00 - 20:00)</span></a>
+            <a class="contact-email" href="mailto:info@company.com">info@company.com</a>
+            <a class="contact-location" href="">USA, <span class="contact-weight-light">San Francisco, 57010, Little Parry St. 826/31</span></a>
+          </figcaption>
+        </figure>
+      </article>
+    </main>
+    <footer>
+      <p>Theme by <span class="footer-name">Pierre Borodin</span> <span class="footer-copyright">&copy; copyright 2012<span></p>
+      <a href="https://www.facebook.com" class="social-icons icon-facebook"></a>
```

Comment:
In this situation (and in the same sets of links used for the team members), I'd probably have added a little extra markup to help out anyone on a screen reader:

```html
<a href="https://www.facebook.com" class="social-icons icon-facebook"><span class="screen-reader-only">Friend us on Facebook</span</a>
```

Where `.screen-reader-only` is hidden text:

```css
.screen-reader-only {
  display: none;
}
```

This adds semantics to the links while not messing with your presentation. Those links, if the CSS is stripped off, are useless without at least a little bit of content.